middlewares/skipAuthenticateForOnboardingExtension.ts

```diff
@@ -0,0 +1,20 @@
+import { NextFunction, Request, Response } from "express"
+import { REQUEST_TYPE } from "../constants/requests";
+
+export const skipAuthenticateForOnboardingExtensionRequest = (authenticate, verifyDiscordBot) => {
```

Comment:
Can we also write a jsDOC comment for this function here?

Reply:
Resolved